roles/collect_modified_images/tasks/parse_image_names.yml

```diff
@@ -21,4 +21,4 @@
 - name: "parse and return image name when timestamp is after start_time"
   set_fact:
     pulled_images: "{{ pulled_images + [image] }}"
-  when: "{{ timestamp > deployment_start_time }}"
+  when: timestamp > deployment_start_time
```

Comment:
This doesn't seem right, does the interpolation not happen when the curly brackets are removed? @samvarankashyap

Reply:
Inside when condition, we need not have the curly braces 
variables can be accessed directly afaik 
@wheelerlaw

Reply:
I see, thank you